modules/oxide/src/less/theme/alien/unknowns.less

```diff
@@ -35,6 +35,7 @@
   .tox-dialog__content-js {
     display: flex;
     flex: 1;
+    max-height: 70vh;
```

Comment:
I don't think this is going to work, because `body-content` has 650px.
https://github.com/tinymce/tinymce/blob/17cb12bb10649b868717e0e42f1cd41a807a9f69/modules/oxide/src/less/theme/components/dialog/dialog.less#L233

The reason `tabpanel` dialogs work and regular `panel` dialogs don't is this resize code:
https://github.com/tinymce/tinymce/blob/3a6455aaebf6034e122ef6cf86beeecef2c7103b/modules/tinymce/src/themes/silver/main/ts/ui/alien/DialogTabHeight.ts#L116-L121

It adjusts the tab height when the window resizes based on the window height minus the height of the dialog "chrome" (header and button rows). Once the height is set, `overflow: auto` kicks in and the scrollbar appears.

Is there a way we can adapt or build equivalent code for regular panels?

Reply:
yes, I saw both, the resize code and that `max-height: 650px`.

About the `max-height: 650px` it is inside the `.tox-dialog__content-js` more specifically in `.tox-dialog__body-content`, so:
  · if the content exceed `650px` it will scroll because of `.tox-dialog__body-content`
  · if the content doesn't exceed `650px` but exceeds the height of the window it will scroll because of `.tox-dialog__content-js`
  · if the content doesn't exceed `650px` and also doesn't exceed the height of the window it will not scroll

About the resize calculation, I think there is a way to do so (I was checking how the height of the panel in tabpanel is calculated), but I thought that a CSS solution would be more solid, but since I'm not a CSS expert I 'd like also to have an opinion from @noxuhax too to understand if my fix is correct and also if solve all via CSS is the best solution in this case or not

Reply:
@noxuhax I removed the `max-height` from `tox-dialog__content-js` and also from `header` and `footer`, and I added that `max-height` with `min` and `calc`, let me know if it is ok